src/config/networks/mainnet.ts

```diff
@@ -22,9 +22,11 @@ const mainnet: NetworkConfig = {
   environment: {
     dev: {
       ...baseConfig,
+      safeUrl: 'https://safe-team.staging.gnosisdev.com/app/',
```

Comment:
This URL is still working (don't know why) but is deprecated. The recommended way is to use the same structure than on the other domains.

`https://safe-team-mainnet.staging.gnosisdev.com/app/`

Same case below

Reply:
Thanks, fixed! ✅